src/controllers/lyrics.js

```diff
@@ -236,6 +241,12 @@ export default function (view) {
 
     view.addEventListener('viewshow', function () {
         Events.on(playbackManager, 'playerchange', onPlayerChange);
+        document.addEventListener('wheel', () => (autoScroll = false));
+        document.addEventListener('keydown', (e) => {
+            if (e.keyCode == 38 || e.keyCode == 40) {
```

Comment:
You can use the functions from `keyboardNavigation` to check the navigation keys:
https://github.com/jellyfin/jellyfin-web/blob/b71aaa05a655242f3cffdc3b46685c9a224f39b7/src/controllers/playback/video/index.js#L1202
https://github.com/jellyfin/jellyfin-web/blob/b71aaa05a655242f3cffdc3b46685c9a224f39b7/src/controllers/playback/video/index.js#L1236